tests/suites/test_suite_bignum_mod.function

```diff
@@ -80,7 +80,196 @@ exit:
 /* END MERGE SLOT 6 */
 
 /* BEGIN MERGE SLOT 7 */
+/* BEGIN_CASE */
+void mpi_residue_setup( )
+{
+    #define RS_ONE       0
+    #define RS_MAX_MIN1  1
+    #define RS_MAX       2
+    const char * s_data[ 3 ] = { "01", "fe", "ff" };
```

Comment:
These should be test data. One test case per value to test.

It's not just a matter of presentation: we need test data with different lengths.